app/views/job_templates/_job_template_callback_tab_headers.html.erb

```diff
@@ -0,0 +1,13 @@
+<li><a id="ansible_tab_header" href="#ansible_callback_enabled" data-toggle="tab"><%= _("Ansible") %></a></li>
+<script type="text/javascript">
+    $(document).ready(function () {
+        var provider_type = $('#job_template_provider_type');
+        provider_type.change(ansibleTabVisibilityByProvider);
+        provider_type.change();
```

Comment:
what does this do?

Reply:
The first call (`provider_type.change(ansibleTabVisibilityByProvider)`) attaches the function to execute when a change event occurs.
The second call (`provider_type.change()`) triggers a change event (to call the `ansibleTabVisibilityByProvider` function when the page loads).